packages/xod-project/src/project.js

```diff
@@ -206,7 +237,16 @@ export const listPatches = def(
 export const listPatchesWithoutBuiltIns = def(
   'listPatches :: Project -> [Patch]',
   R.compose(
-    R.reject(R.compose(isAmong(BUILT_IN_PATCH_PATHS), Patch.getPatchPath)),
+    R.reject(
+      // TODO: extract to separate util?
+      R.compose(
```

Comment:
Another question is why it is called `...WithoutBuiltIns` but returns without builtins *and* terminals? Maybe, rename to `listGenuinePatches` and the util `isGenuinePatchPath`?